webapp/components/Password/Change.spec.js

```diff
@@ -140,11 +142,16 @@ describe('ChangePassword.vue', () => {
 
         describe('mutation rejects', () => {
           beforeEach(() => {
-            // second call will reject
-            wrapper.find('form').trigger('submit')
+            wrapper.find('input#oldPassword').setValue('supersecret')
+            wrapper.find('input#newPassword').setValue('supersecret')
+            wrapper.find('input#confirmPassword').setValue('supersecret')
           })
 
-          it('displays error message', () => {
+          it('displays error message', async () => {
+            await wrapper.find('form').trigger('submit')
+            await wrapper.find('form').trigger('submit')
+            await mocks.$apollo.mutate
+
             expect(mocks.$toast.error).toHaveBeenCalledWith('Ouch!')
           })
         })
```

Comment:
I updated this test as it started failing @roschaefer, but I am not happy with this cause it feels hacky
I was unable to get it to call the `mocks.$toast.error` without 2 submits and a wait for the `mocks.$apollo.mutate`
I think we should remove this test as it is flaky, and test it with cypress
Unlike the `CommentForm` component, I think this would be easy to test because all one needs to do is like I did here where you try to set you newPassword to the same as your oldPassword and expect to see a error toaster.
What do you'all think?

Reply:
I agree that calling `submit` twice is not the way we should solve this. But I certainly would like to have it in place. We certainly should not set the same password in the test since that is a valid option. (How would the server know its the same - its the users fault - we could certainly make a client-side check)